agent/app/agent.go

```diff
@@ -170,12 +167,6 @@ func newAgent(
 	}, nil
 }
 
-// printVersion prints the ECS Agent version string
-func (agent *ecsAgent) printVersion() int {
```

Comment:
It looks like `agent/app/agent_integ_test.go` calls `printVersion`. There is also a windows mock that can probably go away here: `agent/app/agent_windows_test.go`.

Reply:
For the agent_integ_test, I'll replace the call to printVersion with printECSAttributes, now that printVersion doesn't actually depend on newAgent call. 
Removing the mock from agent_window_test.